.github/workflow-templates/readme-template.md

```diff
@@ -9,8 +9,8 @@ Full documentation: link to documentation or wiki
 
 <brief description>
 
-# Example Usage
+## Example Usage
 
-```
+```bash
 <how creator uses it>

```

Comment:
Why the change in the readme template?

Reply:
VSCode Markdown linting always shows me an formatting warning when there are two level 1 headings (lines that begin with one `#`) in the same markdown document. It looks a bit cleaner when there is only one level 1 heading at the top, and all those below are level 2 or lower headings

the `bash` bit is just so that the rendered code has bash syntax highlighting

Just a style thing.

Reply:
Sounds good to me